osgi/karaf/features/src/main/resources/features.xml

```diff
@@ -294,8 +294,8 @@
     <feature name="cxf-tracing-brave" version="${project.version}">
         <feature version="${project.version}">cxf-core</feature>
         <bundle start-level="10" dependency="true">mvn:com.google.code.findbugs/jsr305/${cxf.findbugs.version}</bundle>
-        <bundle start-level="35">mvn:io.zipkin.java/zipkin/${cxf.brave.zipkin.version}</bundle>
-        <bundle start-level="35">mvn:io.zipkin.reporter/zipkin-reporter/${cxf.brave.reporter.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.zipkin2/zipkin/${cxf.brave.zipkin.version}</bundle>
+        <bundle start-level="35">mvn:io.zipkin.reporter2/zipkin-reporter/${cxf.brave.reporter.version}</bundle>
```

Comment:
ps do we need to claim all of these deps in the bundle? or just the top-level (brave)? Seems to add maintenance to have to add each independently, but I don't really know what is required

Reply:
Yeah, we need them otherwise bundle won't start in Karaf, thanks!